covid19_sfbayarea/utils.py

```diff
@@ -28,7 +28,12 @@ def dig(items: Union[Dict[Any, Any], List[Any]], json_path: List[Any]) -> Any:
         raise KeyError(f'Error reading data at path: {json_path}') from error
 
 
-def parse_datetime(date_string: str, timezone: Optional[tzinfo] = PACIFIC_TIME) -> datetime:
+def parse_datetime(
+    date_string: str,
+    timezone: Optional[tzinfo] = PACIFIC_TIME,
+    *,
+    correct_century: bool = True
```

Comment:
Do we need this argument or should we always correct it?

Reply:
I *think* we would always correct, but I wanted it to be there in case we ever needed to deal with some historical data or something, so it would be easy to turn off.